packages/atomic/src/components/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -151,6 +156,13 @@ export class AtomicSearchInterface {
     this.hangingComponentsInitialization.push(event);
   }
 
+  @Listen('atomic/scrollToTop')
+  public scrollToTop() {
+    document
+      .querySelector(this.scrollContainer)
```

Comment:
I think it would be helpful to log a warning if the scrollContainer is not found. e.g.

```
Could not find the scroll container. This will prevent UX interactions that require a scroll from working correctly. Please check the CSS selector in the scrollContainer option
```